README.md

```diff
@@ -134,6 +134,21 @@ $ pip3 install --user git+https://github.com/packit-service/packit.git
 ```
 
 
+### Run from git directly:
+
+You don't need need to install packit, you can run it directly from git (if you have all the dependencies installed):
```

Comment:
```suggestion
You don't need need to install packit to try it. You can run it directly from git (if you have all the dependencies installed):
```
???

Reply:
&&&

Reply:
Sorry... (`???` means that I am not sure..;-)